docs/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The Dafny Language and Software Verification System
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+---
+<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
+<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
+<script type="text/x-mathjax-config">
+        MathJax.Hub.Config({tex2jax: {inlineMath: [['$','$'], ["\\(","\\)"]], displayMath: [ ["$$","$$"], ["\\[","\\]"] ]
+        }});
+</script>
+
+<link rel="stylesheet" href="assets/main.css">
+
+**Quick Links:**
+- [Dafny Reference Manual and User Guide](DafnyRef/DafnyRef)
+- [Index to Dafny Resources for Users](toc)
+- [Dafny GitHub project (for developers of the Dafny tools themselves)](https://github.com/dafny-lang/dafny)
+
+![Dafny Banner](./banner.png "Dafny")
+
+Dafny is a **verification-ready programming language**.
+As you type in your program, Dafny's verifier constantly looks over your shoulder, flags any errors, shows you counterexamples, and congratulates you when your code matches your specifications.
+When you're done, Dafny can **compile your code to C#, Java, JavaScript, Python, C++ or Go** (more to come!), so it can integrate with your existing workflow.
```

Comment:
```suggestion
When you're done, Dafny can **compile your code to C#, Java, JavaScript, C++ or Go** (more to come!), so it can integrate with your existing workflow.
```
The current plan is to announce Python once it's ready with Dafny 4.